cfpb/consumerfinance.gov
# Podcast feed

We'll be releasing the Financial inTuition podcast soon, and we need an RSS feed to go along with it. For more info (including why we added this to the site root), see GHE/CFPB/Consumer-Facing-Iteration/issues/76.

---

## Additions

- `financial-intuition.xml` to the site root

## How to test this PR

See GHE/CFPB/Consumer-Facing-Iteration/issues/76 for full testing instructions. You can also make sure the feed contains all the required and recommended elements [recommended by Apple](https://help.apple.com/itc/podcasts_connect/#/itcb54353390).

When reviewing the XML, note that I had to keep text like descriptions and titles on one line to avoid weird line breaks in podcast apps, so it doesn't follow our usual 80-character line limit.

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
